### ISSUE TITLE
different results in matlab and C version?

### ISSUE THREAD

**Issue by Thanh-Binh** [yashdv/Image-search#1]
I have just tested them under Ubuntu and found that they provide different results of GIST features by the same input image.
Do you have any idea?
Thanks

**Comment by yashdv** [comment]
Yeah, I would expect them to be different because the Matlab GIST is a different implementation than the C GIST. They are not identical.

**Comment by Thanh-Binh** [comment]
I do not think so! From my quick view, the c version seems to be ported directly from matlab code!

**Comment by Thanh-Binh** [comment]
Do you know the min and max value of a GIST feature value?

**Comment by yashdv** [comment]
Hmm, not sure. As you can see the GIST codes have been borrowed from the INRIA lab.
